app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,5 @@
     <string name="leaderboard">Leaderboard</string>
     <string name="play">Play</string>
     <string name="settings">Settings</string>
+  <string name="developer_info">Created by Mukola Huk &amp; PanVova</string>
```

Comment:
xD

Reply:
> Placing menu code in MainActivity looks weird, I thought we use single activity approach, so there should be separate fragment for menu

Yes, you are right but the menu fragment was created in other PR so it would be useless to create another module here and then merge it. I can guess it would be better to fix small PR problems